src/index.html

```diff
@@ -6,25 +6,85 @@
     <title>Leaderboard</title>
```

Comment:
- [ ] In your readme file the live link does not represent your current project. Kindly, update  it!

Reply:
I build the project, the dist has now updated.